grails-app/conf/application.yml

```diff
@@ -271,7 +271,8 @@ environments:
             logSlowQueries: false
             username: "openboxes"
             password: "openboxes"
-            url: jdbc:mysql://localhost:3306/openboxes?serverTimezone=UTC&useSSL=false
+            url: jdbc:tc:mysql:///openboxes?serverTimezone=UTC&useSSL=false  # https://java.testcontainers.org/modules/databases/jdbc/
```

Comment:
- Move comemnt above 
- See if settings are actually getting applied here (try useSSL=true or some obvious setting)
- If the settings aren't getting passed through to the driver then we should set them in a mysql cnf override file as specified in docs (https://java.testcontainers.org/modules/databases/mysql/)
- Find out what is the default MySQL test container version and add that as either a comment or explicit 
- Write docs if we figure out how to actually get into the test container

Reply:
* Test whether we can override configuration via env variables and system properties
   * export DATASOURCE_URL = 
   * grails -DdataSource.url= test-app
* If it doesn't work, then we need to do some passthrough shinanegans with gradle to make it work properly https://stackoverflow.com/questions/28196042/how-override-grails-configuration-from-command-line

Reply:
With the config I want to 

1. make it work out of the box without any changes
2. use passthrough in build.gradle 
3. do something like this in application.yml
```
config:
    property: System.get("config.property", defaultValue)
```
I would prefer 1 or 2, and would like to stay away from 3 like the plague because it would require a code change to application.yml every time we add a new property.